pkg/database/rql/UserModel.go

```diff
@@ -0,0 +1,124 @@
+package rql
+
+import (
+	"fmt"
+	"github.com/quan-to/chevron/pkg/models"
+	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).